runtime-modules/proposals/engine/src/lib.rs

```diff
@@ -822,12 +822,8 @@ impl<T: Trait> Module<T> {
 
                 let now = Self::current_block();
 
-                // If exact block is set - cancel the grace period.
-                let ready_for_execution = if proposal.exact_execution_block.is_none() {
-                    proposal.is_grace_period_expired(now)
-                } else {
-                    proposal.is_execution_block_reached(now)
-                };
+                let ready_for_execution = proposal.is_grace_period_expired(now)
```

Comment:
Can we replace with `Proposal::is_ready_for_execution`.

Reply:
Done